src/components/BreedsList.vue

```diff
@@ -49,10 +58,35 @@ export default {
         .then((results) => {
           this.isLoading = false;
           if (results.length > 0) {
+            console.log(results);
+            this.breedsList = results;
+            this.$emit("reload", this.breedsList);
```

Comment:
zakaj je pa ta emit event?

Reply:
to je napaka, sem testiral kako bi povezal reload gumb s tem da se ponovno naložijo vsi podatki.